### DISCUSSION TITLE
IOS cannot verify that the app requires an internet connection to verify if the developer is trusted

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2999] [Q&A]
![微信截图_20240108185404](https://github.com/airsdk/Adobe-Runtime-Support/assets/91530077/d9825414-0be8-4024-b3f2-f2ebb606f89f)
My iPad system version is iOS 15.5, which can be installed normally when I publish using Air SDK, but this verification prompt will be raised.
I have tried to remove the UseNativeCodesign=true annotation from adt.cfg, but it still doesn't work well after being published. The Apple certificate has also been regenerated and downloaded.
Strangely, some iPads can be installed and used normally. Did I overlook something?
Thank you!

-------------------------------------------------------------
To add, I am using AIR 50.2.4.2 (19-Dec-2023), and the iPad iOS versions that can be installed are all below 15.5, with 15.3 and 15.4, both of which can be installed normally.
There is another issue, which is that I can 
install the previously packaged IPA file on this device and open it normally. It's just that my newly packaged installation will prompt this issue.

**Comment by Jason0247** [comment]
[files.zip](https://github.com/airsdk/Adobe-Runtime-Support/files/13860177/files.zip)
These are two IPA files, one is useful and the other is not.

**Reply by ajwfrost** [reply]
Hi

What sort of application/certificate is it? The below is a hit we get from Apple support for this error message, it sounds like it's expecting a corporate certificate to be present already on the device (enterprise licensing)?
https://discussions.apple.com/thread/254336444?sortBy=best

thanks

**Reply by Jason0247** [reply]
I am using a personal certificate, packaged in App Store and Ad Hoc format, not a corporate certificate

**Reply by Jason0247** [reply]
Enterprise certificates can be trusted in the settings of the iPad, but personal certificates do not have such a setting.
I don't know why, the previously released iPad can still be installed and opened normally, while the newly packaged one still has the same problem even if I use the latest SDK.

**Reply by Jason0247** [reply]
https://discussions.apple.com/thread/254336444?sortBy=best
I have read the reply to this link and the answer provided is for the enterprise certificate.
But this question should be the same as my situation, using personal certificates.
